HollowEngine/BulletHell/src/DungeonGeneration/DungeonManager.cpp

```diff
@@ -46,6 +49,14 @@ namespace BulletHell
     {
         mSeed = seed;
     }
+	
+    void DungeonManager::Construct()
```

Comment:
Note for later, add int floor index that can be obtained from game logic manager. Hard coded to index 0 right now